src/lib/jsonParseV2/parseSubscription.cpp

```diff
@@ -462,73 +380,172 @@ static std::string parseEntitiesVector(ConnectionInfo* ciP, EntityIdVector* eivP
+  {
+    return error(ciP, "notification has http and httpExtended");
+  }
+  else if (notification.HasMember("http"))
   {
     const Value& http = notification["http"];
     if (!http.IsObject())
     {
-      OrionError oe(SccBadRequest, "http notitication is not an object");
+      return error(ciP, "http notification is not an object");
+    }
+
+    // http - url
+    {
+      Opt<std::string> urlOpt = getStringMust(http, "url", "url http notification");
 
-      alarmMgr.badInput(clientIp, "http notification is not an object");
-      return oe.render(ciP, "");
+      if (!urlOpt.ok())
+      {
+        return error(ciP, urlOpt.error);
+      }
 
+      {
+        std::string  host;
+        int          port;
+        std::string  path;
+        std::string  protocol;
+
+        if (parseUrl(urlOpt.value, host, port, path, protocol) == false)
+        {
+          return error(ciP, "Invalid URL parsing notification url");
+        }
+      }
+      subsP->notification.httpInfo.url =      urlOpt.value;
+      subsP->notification.httpInfo.extended = false;
     }
 
-    if (!http.HasMember("url"))
+  }
+  else if (notification.HasMember("httpExtended"))
+  {
+    const Value& httpExt = notification["httpExtended"];
+    if (!httpExt.IsObject())
+    {
+      return error(ciP, "httpExtended notification is not an object");
+    }
+
+    // URL
     {
-      alarmMgr.badInput(clientIp, "url http notification is missing");
-      OrionError oe(SccBadRequest, "url http notification is missing");
+      Opt<std::string> urlOpt = getStringMust(httpExt, "url", "url httpExtended notification");
+
+      if (!urlOpt.ok())
+      {
+        return error(ciP, urlOpt.error);
+      }
 
-      return oe.render(ciP, "");
+      subsP->notification.httpInfo.url = urlOpt.value;
     }
-    const Value& url = http["url"];
-    if (!url.IsString())
+
+    // method -> verb
     {
-      alarmMgr.badInput(clientIp, "url http notification is not a string");
-      OrionError oe(SccBadRequest, "url http notification is not a string");
 
-      return oe.render(ciP, "");
+      Opt<std::string> methodOpt = getStringMust(httpExt, "method", "method httpExtended notification");
+
+      if (!methodOpt.ok())
+      {
+        return error(ciP, methodOpt.error);
+      }
+      Verb  verb         = str2Verb(methodOpt.value);
+      /*
+       *  CHECK IT IS VALID
+       */
+
+      subsP->notification.httpInfo.verb = verb;
+    }
+
+    // payload
+    {
+      Opt<std::string> payloadOpt = getStringMust(httpExt, "payload", "payload httpExtended notification");
+      if (!payloadOpt.ok())
+      {
+        return error(ciP, payloadOpt.error);
+      }
+      subsP->notification.httpInfo.payload = payloadOpt.value;
+    }
+
+
+    // qs
+    if (httpExt.HasMember("qs"))
+    {
+      const Value& qs = notification["qs"];
+      if (!qs.IsObject())
+      {
+        return error(ciP, "notificaction httpExt qs is not an object");
```

Comment:
If httpExt refers to "the extended alternative to the `http` field in `notificationsp" the right name is `httpExtended` (see https://github.com/telefonicaid/fiware-orion/pull/2125).

Several ocurrences of `httpExt`. Overall search should be done.